app/javascript/components/applications/ApplicationView.jsx

```diff
@@ -84,20 +83,18 @@ class ApplicationView extends React.Component {
     displayButtons() {
         let buttons = null;
         if (this.props.organization) {
-            console.log('org signed in');
             if (this.props.application.status === null || this.props.application.status === "pending") {
                 buttons = (
-                    <div>
-                    <button onClick={this.handleInterview}>Interview</button>
-                    <button onClick={this.handleDeny}>Reject</button>
+                    <div className="mv4">
+                    <button className="accept-button f5 pa2 fr" onClick={this.handleAccept}>Interview</button>
```

Comment:
Make sure this calls the right function. I think this should be `handleInterview`

Reply:
Sorry my bad,  but also it looks like this code isn't updated (no className).